### DISCUSSION TITLE
Fallo en la mayoría de los sites

### DISCUSSION THREAD

**Discussion by cortex1234567** [Oihalitz/PerubianBot#1] [Q&A] [answered]
Hola shur,

La mayoría de los sites me fallan (skipped) hay que hacer algo más allá de incluir los datos del sujeto pasivo? Saludos

**Comment by TFujiwara** [comment, marked as answer]
Parece que falla en algunos sitios por que muchos piden prefijo internacional de número. A ver si el shur o alguien lo puede arreglar y hacer una PR.
